luz_asm_sim/lib/simlib/interactive_cli.py

```diff
@@ -44,7 +44,16 @@ def show_memory(sim, addr):
         printme('\n')
 
 
-COMMANDS = ('s', 'r', 'sr', 'm', 'rst', 'help', 'q', 'set')
+def show_disassembly(sim, first_address, num, replace_alias):
```

Comment:
I also switched the print() function to printme() for consistency with the rest of the code.